src/cascadia/TerminalApp/MinMaxCloseControl.xaml

```diff
@@ -135,6 +135,8 @@ the MIT License. See LICENSE in the project root for license information. -->
             x:Name="MinimizeButton"
             x:Uid="WindowMinimizeButton"
             Style="{StaticResource CaptionButton}"
+            ToolTipService.ToolTip="Minimize"

```

Comment:
Moving my comment from the original thread: These strings ("Minimize", "Maximize", "Close") probably all need to be coming from our [Resources.resw](https://github.com/microsoft/terminal/blob/master/src/cascadia/TerminalApp/Resources/en-US/Resources.resw) file, and used here as `{StaticResource WhateverTheResourceNameIs}` so that the strings can be localized into other languages. 

I'm _sure_ there's an example somewhere else in this repo, but I'm not sure we actually have any other XAML tooltips so that's not the best place to go looking 😅

Reply:
> Moving my comment from the original thread: These strings ("Minimize", "Maximize", "Close") probably all need to be coming from our [Resources.resw](https://github.com/microsoft/terminal/blob/master/src/cascadia/TerminalApp/Resources/en-US/Resources.resw) file, and used here as `{StaticResource WhateverTheResourceNameIs}` so that the strings can be localized into other languages.
> 
> I'm _sure_ there's an example somewhere else in this repo, but I'm not sure we actually have any other XAML tooltips so that's not the best place to go looking 😅
Sure, maybe you can implement that?

Reply:
Are you only talking about the `minmaxclose control` or `tabrowcontrol`.xaml or both?

Reply:
Also, where can I find the menu code/design? I can't seem to find it in visual studio's solution explorer.

Reply:
Thank you @zadjii-msft. The newest commits are concerning your feedback (changes made to `resources.resw`)